File: abc/abc168/abc168_a.cs
// https://atcoder.jp/contests/abc168/submissions/13388935

using System;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine()) % 10;
        if (N == 3)
        {
            Console.WriteLine("bon");
        }
        else if (N == 0 || N == 1 || N == 6 || N == 8)
        {
            Console.WriteLine("pon");
        }
        else
        {
            Console.WriteLine("hon");
        }
    }
}

File: abc/abc168/abc168_a.fs
// https://atcoder.jp/contests/abc168/submissions/13388975

stdin.ReadLine()
|> int
|> fun N ->
    match N % 10 with
    | 3 -> "bon"
    | 0
    | 1
    | 6
    | 8 -> "pon"
    | _ -> "hon"
    |> stdout.WriteLine
